index.html

```diff
@@ -9,10 +9,22 @@
     <link rel="stylesheet" href="./css/home.css">
 </head>
 <body>
-    <!-- HEADER START
-    HEADER END
-
-    HERO START
+    <header>
+        <img class="logo" src="./img/name-logo.png" alt="logo">
+        <nav>
+            <ul class="nav-links">
+                <li><a href="#">home</a></li>
+                <li><a href="#">pricing</a></li>
+                <li><a href="#">career</a></li>
+                <li><a href="#">pages</a></li>
+                <li><a href="#">blog</a></li>
+                <li><a href="#">contact</a></li>
+            </ul>
```

Comment:
po ul tag'o turėtum įsidėti mobile-menu kaip elementą kurio `display: none`.  Tuomet reikia media query nurodyt, kad pasiekus tam tikrą ekrano dydį jo display jau yra matomas `pvz. display: block`, tuo tarpu  ul tampa nebematomas `display: none` ar `visibility: hidden`. Tai pasiekus naudojam JS event handlerius, kad on click rodytų menu

Reply:
Elementai nematomi patampa pagal css/components/header.css 46-50 eilutes.

Reply:
Šaunu! Tai dar mobile menu pridėt su funkcionavimu. Štai labai panašus pvz. kaip net be JS galima padaryt https://codepen.io/shieldsma91/pen/zLpbLX